test/jobs/autotitle_conversation_job_test.rb

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
+class AutotitleConversationJobTest < ActiveJob::TestCase
+  test "sets conversation title to 'Hear me'" do
+    ChatCompletionAPI.stubs(:get_next_response).returns({"topic" => "Hear me"})
+
+    conversation = conversations(:greeting)
+    AutotitleConversationJob.perform_now(conversation.id)
+    assert_equal "Hear me", conversation.reload.title
```

Comment:
The minitest syntax for this is:

```suggestion
    ChatCompletionAPI.stub :get_next_response, {"topic" => "Hear me"} do
      conversation = conversations(:greeting)
      AutotitleConversationJob.perform_now(conversation.id)
      assert_equal "Hear me", conversation.reload.title
    end
```